resources/benchmark-runner.mjs

```diff
@@ -118,6 +119,11 @@ export class BenchmarkRunner {
         this._suites = suites;
         this._client = client;
         this._page = null;
+        this._metrics = {
+            __proto__: null,
+            Total: new Metric("Total"),
+            Score: new Metric("Score", "rpm"),
```

Comment:
Why are we using "rpm" here? Didn't we decide to move away from it?

Reply:
Ups, copy-pasta. Changed unit to "score" as well.